src/sortable.h

```diff
@@ -94,13 +53,9 @@ RSSortingVector *NewSortingVector(int len);
 /* Free a sorting vector */
 void SortingVector_Free(RSSortingVector *v);
 
-/* Save a document's sorting vector into an rdb dump */
-void SortingVector_RdbSave(RedisModuleIO *rdb, RSSortingVector *v);
```

Comment:
not needed anymore? RDB breaking change?

Reply:
This function was not in use. We don't save the sorting vectors to the RDB since RediSearch 2.0.